cmd/clusterctl/client/cluster/cert_manager_test.go

```diff
@@ -35,22 +34,15 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	clusterctlv1 "sigs.k8s.io/cluster-api/cmd/clusterctl/api/v1alpha3"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/client/config"
-	manifests "sigs.k8s.io/cluster-api/cmd/clusterctl/config"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/internal/scheme"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/internal/test"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func Test_VersionMarkerUpToDate(t *testing.T) {
```

Comment:
I think we should preserve this test, even in a slightly different version

The test should call newCertMangerClient and check that version and hash are set; this makes sure that whenever we are upgrading the cert-manager manifest, the new manifest should abide the same contract